msg/versioned/BatteryStatus.msg

```diff
@@ -22,9 +22,9 @@ uint8 cell_count           # [-] [@invalid 0] Number of cells
 
 
 uint8 source                   # [@enum SOURCE] Battery source
-uint8 SOURCE_POWER_MODULE = 0  # Power module
-uint8 SOURCE_EXTERNAL = 1      # External
-uint8 SOURCE_ESCS = 2          # ESCs
+uint8 SOURCE_POWER_MODULE = 0  # Power module (analog ADC or I2C power monitor)
+uint8 SOURCE_EXTERNAL = 1      # External (MAVLink or external driver)
```

Comment:
Would a CAN power module be a "Power module" or External driver?

Reply:
I added enough info to indicate that CAN is part of external. I haven't captured that you need to enable the subscriber, but that is part of the CAN setup.